pkg/imagebuilder/content.go

```diff
@@ -195,6 +202,7 @@ func (c ConfigBuilder) templateString(name string, text string) (string, error)
 		"ServiceBaseURL":  c.serviceBaseURL,
 		"infraEnvId":      c.infraEnvID,
 		"PullSecretToken": c.pullSecretToken,
+		"NodeZeroIP":      c.nodeZeroIP,
```

Comment:
Note that this actually needs to be a _host_ (i.e. if it is an IPv6 address the IP must be surrounded by []).
We will need to support IPv6 at some point, and will have a lot less pain if we are clear about what is an IP and what is part of a URL as we go.